parallel-trials/experiment.yaml

```diff
@@ -43,16 +47,15 @@ spec:
   namespaceSelector:
     matchLabels:
       experiment: parallel-postgres
-  replicas: 4
-  template: # trial
+  trialTemplate:
     spec:
-      template: # job
+      jobTemplate:
         spec:
-          template: # pod
+          template:
             spec:
               containers:
-              - image: crunchydata/crunchy-pgbench:centos7-11.4-2.4.1
-                name: pgbench
+              - name: pgbench
+                image: crunchydata/crunchy-pgbench:centos7-11.4-2.4.1
```

Comment:
We'll probably want to pull in the changes from #54  here, not sure I realized it was here as well ( thought this was just jvm ).

Reply:
Happy to pull in #54 here

Reply:
Parallel trials being dropped, this no longer applies.